### DISCUSSION TITLE
Any good sources for a tutorial on setting up on Windows?

### DISCUSSION THREAD

**Discussion by bobloadmire** [drakkan/sftpgo#1653] [Q&A]
I have installed SFPTGo on my windows machine, I can login to admin locally, but I cannot connect to the SFTP server on localip:2022 with the user I created. I feel like I'm missing something, but I have no clue what it is. My ftp client is returning: "exhausted available authentication methods" I installed another FTP server called Porta SFTP and it works great, but I want to use SFTPGo, but it appears the out of the box config needs more customization before it works, but i'm kinda lost. Thanks.

**Comment by drakkan** [comment]
Hello, Windows or Linux makes no difference, after you install SFTPGo it listens on port 2022 by default. Please check your logs to understand what is going on.

**Reply by bobloadmire** [reply]
I treid that, I have no idea how to do that. the logs section in the howto gives no instruction on how to access the logs: https://sftpgo.github.io/latest/logs/

there's nothing in the webgui for logs, so I have no idea what to do.

**Reply by drakkan** [reply]
> I treid that, I have no idea how to do that. the logs section in the howto gives no instruction on how to access the logs: https://sftpgo.github.io/latest/logs/
> 
> there's nothing in the webgui for logs, so I have no idea what to do.

The location of the log file depends on the installation mode and the OS. On Windows the logs are in `C:\ProgramData\SFTPGo\logs`
